recipes/blockbuster/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+make
+mkdir $PREFIX/bin
+cp $SRC_DIR/blockbuster.x $PREFIX/bin
```

Comment:
@johanneskoester as far as I remember krona is more complicated. It just symlinks from the scripts directory to `PREFIX/bin` and the scripts dir needs to be next to the lib dir. So either I move everything into $PREFIX (with lib and share) or I create a new subdirectory.